saw-remote-api/python/tests/saw-in-progress/drbg/drbg.py

```diff
@@ -308,14 +300,15 @@ def specification(self):
 llvm_verify(mod, "s2n_drbg_bytes_used", bytes_used_spec())
 
 
-blk_enc_ov = llvm_verify(mod, "s2n_drbg_block_encrypt", contract = block_encrypt_spec(), lemmas = [encryptUpdate_ov], script = yices(unints=['block_encrypt']))
+blk_enc_ov = llvm_verify(mod, "s2n_drbg_block_encrypt", contract = block_encrypt_spec(),
+        lemmas = [encryptUpdate_ov], script = ProofScript([yices(unints=['block_encrypt'])]))
```

Comment:
I'm somewhat surprised that `yices(unints=['block_encrypt'])` was changed to `ProofScript([yices(unints=['block_encrypt'])])`. Is there a reason to favor the latter (which is more verbose)?

Reply:
I didn't make the change, but at a glance it appears the former did not match the [type signature](https://github.com/GaloisInc/saw-script/blob/2f7645a651208151e5531e36887f625695d3456e/saw-remote-api/python/saw_client/__init__.py#L565) for `llvm_verify` but the latter does, but it worked anyway because the type signature was more restrictive than it needed to be (i.e., `ProofTactic` has a `to_json` method that will produce JSON of the correct form as well).

We could tweak the `script` argument's type to just be more flexible and allow several things that have an obvious semantic meaning when interpreted as a `script` (e.g., we could allow either a `ProofTactic`, `List[ProofTactic]`, or `ProofScript` and add some logic to handle each case appropriately).

Reply:
Hm, I didn't realize that `ProofTactic` and `ProofScript` were two separate types. I suppose that makes sense, and looking at the code, [this isn't already an established convention](https://github.com/GaloisInc/saw-script/blob/65d764b0e5cc0f4608b75448354e8c47f06a8cdc/saw-remote-api/src/SAWServer/ProofScript.hs#L79). I suppose I was just lucky that my cavalier approach of always writing `script = yices(...)` happened to always work. Sorry for the noise!